### DISCUSSION TITLE
Nextjs loop through all available API routes

### DISCUSSION THREAD

**Discussion by Prottoy2938** [vercel/next.js#14299] [Help] [answered]
**Nextjs** creates API endpoints from `pages/api/` directory. Is it possible to find how many available API's are there and using a loop to send request to all of them?

**Comment by cristiand391** [comment, marked as answer]
Yep, you can use the file system API to read all the filenames and create an url for each endpoint.

I've written an example here, feel free to check it:
https://github.com/cristiand391/nextjs-loop-api-endpoints
